genesyscloud/routing_queue/resource_genesyscloud_routing_queue_schema.go

```diff
@@ -292,11 +292,10 @@ func ResourceRoutingQueue() *schema.Resource {
 				},
 			},
 			"conditional_group_routing_rules": {
-				Description: "The Conditional Group Routing settings for the queue.",
+				Description: "The Conditional Group Routing settings for the queue. **Note**: conditional_group_routing_rules is deprecated in genesyscloud_routing_queue. CGR is now a standalone resource, please set ENABLE_STANDALONE_CGR in your environment variables to enable and use genesyscloud_routing_queue_conditional_group_routing",
 				Type:        schema.TypeList,
 				Optional:    true,
 				MaxItems:    5,
-				Deprecated:  "conditional_group_routing_rules is deprecated in genesyscloud_routing_queue. CGR is now a standalone resource, please set ENABLE_STANDALONE_CGR in your environment variables to enable and use genesyscloud_routing_queue_conditional_group_routing",
```

Comment:
Why did you remove the `Deprecated` field and merge it into the Description?

Reply:
We noticed it was generating a huge amount of warning messages when AirBnB was running their jobs.  Their jobs are unexpectedly crashing so we are trying to see filter the noise (and a potential source of slowness because all of the IO)